app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -87,6 +88,10 @@ public function store(StoreIntegration $storeIntegration): RedirectResponse
 
         $this->integrationRepository->save($integration);
 
-        return Redirect::route('integrations.index');
+        $language = $storeIntegration->headers->get('Accept-Language') ?? 'en';
```

Comment:
I see a mix of "default" languages. Here it is `en` and [here](https://github.com/cultuurnet/publiq-platform/pull/342/files#diff-a211f19734559dfaf8c7897fc66882ef1b6855b8ce9026e9226b43b796cd963dR33) it is `nl`.

Maybe add the default language as a method on `TranslatedRoute`?